inst/metadata/advanced_run_options.json

```diff
@@ -214,6 +214,36 @@
 					"helpText": "t_(OPTIONS_ADVANCED_LOGIT_HELP)"
 				}
 			]
+		},
+	        {
+			"label": "Approximate district survey design effect",
```

Comment:
Can you move all the strings into the resource bundle https://github.com/mrc-ide/naomi/blob/master/inst/traduire/en-translation.json#L78

This is for each string which gets returned to the user (and so requires translation) pick a key name and use that here e.g.
```
"label": "t_(OPTIONS_DESIGN_EFFECT_LABEL)"
```

Then there should be a line in the resource bundle
```
"OPTIONS_DESIGN_EFFECT_LABEL": "Approximate district survey design effect"
```

That should be for all user facing strings here

Reply:
Thanks. Done in 2efcab9.

At some point I thought Rich mentioned something like that everything that goes into the translation resource needs to be tested. Do I need to do anything further on this?

Reply:
I don't know about that - everything there needs to be translated but I don't believe we're testing every string at the moment. Maybe @richfitz can comment? 

This looks fine to me - I will ask Anne to give us a translation of these strings and add that to this PR before merging.